### PR TITLE
Specify registry-url and scope in CD workflow

### LINKED ISSUE #349: CD Workflow failing
**Description**
The CD workflow present in this repo which is based on the official GitHub Actions documentation is failing when a new release is published with the following error:
`npm ERR! 401 Unauthorized - PUT https://registry.npmjs.org/@creativecommons%2fvocabulary - You must be logged in to publish packages.`
On searching the web for the possible reasons for this error, I found that since there are two package repositories, namely `npm` and `GitHub Packages` available, and also since Vocabulary is a scoped package, so both the `registry-url` and `scope` has to be explicitly specified in the workflow.

**Reproduction**

The error can be observed by visiting [this link.](https://github.com/creativecommons/fonts/runs/622557954?check_suite_focus=true#step:5:197)

**Expected behavior**

The workflow shouldn't fail.

**Additional context**

The problem may be resolved by specifying the `registry-url` and the `scope` in the workflow.

**Want to take up?**

Yes

### PR DESCRIPTION
**Pull request by SaurabhAgarwala**
## Fixes

Fixes #349 by @SaurabhAgarwala 

## Description

The CD workflow currently present is failing. On looking into the possible solutions for the problem, specifying the `registry-url` and the `scope` may solve the problem. This PR adds both of these parameters.

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by SaurabhAgarwala** [comment]
Please review and test it @brenoferreira @dhruvkb whenever possible.

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -20,6 +20,8 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: '12.16.1'
+        registry-url: https://registry.npmjs.org
+        scope: creativecommons
```
Comment (dhruvkb) [inline .github/workflows/cd.yml]: Are these needed? As far as I can tell from the [`action.yml` file](https://github.com/actions/setup-node/blob/master/action.yml), both are optional fields.

Comment (dhruvkb) [reply .github/workflows/cd.yml]: We're not using a scoped registry, `@creativecommons/vocabulary` is hosted on the public `npm` registry.

Comment (SaurabhAgarwala) [reply .github/workflows/cd.yml]: I looked into many examples for CD, and in all of them I found the `registry-url` specified. Also, as specified in the link shared by you, I think this field will be requried in our case for authenticating and reading from `env.NODE_AUTH_TOKEN`. From what I understand, it is optional in cases when we don't need to set up the registry for auth in the project-level `.npmrc` file.

Comment (SaurabhAgarwala) [reply .github/workflows/cd.yml]: > We're not using a scoped registry, `@creativecommons/vocabulary` is hosted on the public `npm` registry.

Please correct me if I am wrong. From what I understand from this [npm documentation](https://docs.npmjs.com/using-npm/scope.html), scopes are needed for grouping the packages of an organization. And as we refer this Vocabulary project in npm as `@creativecommons/vocabulary` (both in [URL](https://www.npmjs.com/package/@creativecommons/vocabulary) and while installing it) so this means `creativecommons` will be it's scope name. Also, I find [another package](https://www.npmjs.com/package/vocabulary) in npm by the same name as vocabulary, which I doesn't have any scope as it directly referred to as `vocabulary` both in it's URL as well as while installing.

Comment (SaurabhAgarwala) [reply .github/workflows/cd.yml]: I am not being able to find any other possible reason for the error currently, so if we can just try it and see if it works. If it doesn't work even after this, then I will look into it further.

Comment (dhruvkb) [reply .github/workflows/cd.yml]: It makes sense, we'll merge it before the next release is about to come out.

**Review by dhruvkb — APPROVED** [review]

**Review by dhruvkb — CHANGES_REQUESTED** [review]
@SaurabhAgarwala on further thought, can you move it to `creativecommons/fonts`? It's a much lower risk repository so we can make multiple iterations there.

**Comment by SaurabhAgarwala** [comment]
> @SaurabhAgarwala on further thought, can you move it to `creativecommons/fonts`? It's a much lower risk repository so we can make multiple iterations there.

Yeah, definitely @dhruvkb. That's a great suggestion. I will make the corresponding changes in the `fonts` repo soon, and make a PR for it.

**Comment by dhruvkb** [comment]
Blocked while we test creativecommons/fonts#83 as a resolution of creativecommons/fonts#82.

**Comment by SaurabhAgarwala** [comment]
@dhruvkb I have made the required changes

**Review by dhruvkb — APPROVED** [review]
Should work like creativecommons/fonts#86.

**Pull request merged**